src/tag/examples.js

```diff
@@ -59,12 +70,18 @@ export default {
             <Tag
               key={color}
               color={color}
```

Comment:
I thought we're getting rid of the `color` prop in favor of `kind`. But we can delete this prop in a follow-up update if everyone's on board. What do you think?

Reply:
That was my plan, but getting rid of color makes basic overrides more complicated than necessary.